linked-courses-importers.md

```diff
@@ -0,0 +1,214 @@
+# Linked Courses importers and commands
+
+This documentation is written to help you decide what to use if you'd like to setup your own application instance of Linked Courses.
+
+#
+
+<!-- vim-markdown-toc GFM -->
+
+* [YSO - General Finnish ontology **(Required)**](#yso---general-finnish-ontology-required)
+  * [What is it?](#what-is-it)
+  * [What depends on it?](#what-depends-on-it)
+  * [How to use it?](#how-to-use-it)
+* [tprek - Location registry **(Required)**](#tprek---location-registry-required)
+  * [What is it?](#what-is-it-1)
+  * [What depends on it?](#what-depends-on-it-1)
+  * [How to use it?](#how-to-use-it-1)
+* [helmet - Helsinki Metropolitan Area Libraries](#helmet---helsinki-metropolitan-area-libraries)
+  * [What is it?](#what-is-it-2)
+  * [What depends on it?](#what-depends-on-it-2)
+  * [How to use it?](#how-to-use-it-2)
+* [install_templates](#install_templates)
+  * [What is it?](#what-is-it-3)
+  * [What depends on it?](#what-depends-on-it-3)
+  * [How to use it?](#how-to-use-it-3)
+* [geo_import](#geo_import)
+  * [What is it?](#what-is-it-4)
+  * [What depends on it?](#what-depends-on-it-4)
+  * [How to use it?](#how-to-use-it-4)
+
+<!-- vim-markdown-toc -->
+
+## YSO - General Finnish ontology **(Required)**
+
+### What is it?
+First, some terminology is in order. Starting with *ontology*:
+
+> In computer science and information science, an ontology encompasses a representation,
+> formal naming and definition of the categories, properties and relations between the
+> concepts, data and entities that substantiate one, many or all domains of discourse.
+>
+> From [Wikipedia of Ontology (information science)](https://en.wikipedia.org/wiki/Ontology_(information_science))
+
+YSO, short for *Yleinen suomalainen ontologia*, is basically a +30,000 collection of **concepts** in Finnish culture, translated in 3 languages.
+
+> General Finnish Ontology YSO is a trilingual ontology consisting mainly of general concepts.
+> YSO has been founded on the basis of concepts in Finnish cultural sphere.
+>
+> From https://finto.fi/yso/en/. You can browse the concepts there if you wish
+
+Basic example would be, let's say a Portuguese language teacher coming to Helsinki to teach Portuguese to children.
+
+We might categorize it with "education", "Portuguese", and "fun" *concepts*.
```

Comment:
I would definitely add the category "children" in the example, it is by far the most common keyword in Linkedevents actually, with 46 000+ children's events :D (http://api.hel.fi/linkedevents/v1/keyword/ shows it first in list)

Reply:
Sure. Done in 85b0ba25e6b037d2145451e8387be664819d92b5